lib/uppy/job/use_case.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Uppy.Job.UseCase do
+  import Ecto.Changeset
+
+  alias Uppy.{Repo, Job}
+
+  @required_fields ~w(url)a
+  @url_format ~r/^http(s)?:\/\/.(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&\/\\=]*)$/i
```

Comment:
It's not the same thing, but seeing this regex reminded me of https://davidcel.is/articles/stop-validating-email-addresses-with-regex/.

Maybe a simpler regex is fine here?

```suggestion
  @url_format ~r'^http(?:s)?://\S+\.\S+$'i
```

Or if you feel strongly about validating it, take a look at https://mathiasbynens.be/demo/url-regex which lists a bunch of weird cases.

Reply:
I remember our discussion from a year ago, and I still agree with you. I've updated the PR to your suggestion 9fa6fba.